app-sso/concepts/levels-of-consumption.hbs.md

```diff
@@ -0,0 +1,287 @@
+# The three levels of AppSSO consumption
+
+This topic describes the different levels of consuming AppSSO services and
+explains when and why you might choose to use one level over the other.
+
+Generally, the recommendation is to consume AppSSO through `ClassClaim` but
+there can be situations where the lower level `WorkloadRegistration` or
+`ClientRegistration` are a better fit.
+
+At its core consumption of AppSSO is about obtaining client credentials for an
+authorisation server and loading them into a running workload. In particular, it
```

Comment:
[nit]: in the rest of the documentation we used the American spelling of `authorization`. I don't mind which one we go with, but I'd prefer to be consistent.

Reply:
ack ty